analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -77,9 +69,33 @@ class Gradle(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConfig
                 Gradle(analyzerConfig, repoConfig)
     }
 
-    val maven = MavenSupport { localRepositoryManager ->
-        GradleLocalRepositoryManager(localRepositoryManager)
-    }
+    val maven = MavenSupport(object : WorkspaceReader {
```

Comment:
Moving this object to a separate class `GradleCacheReader` would make the code testable.

Reply:
Done (as a nested class, similar to `GradleLocalRepositoryManager` before).